functions/helpers.bash

```diff
@@ -479,7 +479,8 @@ wait_for_apt_to_be_ready() {
 ##    apt_update()
 ##
 apt_update() {
-  nohup apt-get update &> /dev/null & export PID_APT=$!
+  #nohup 
```

Comment:
why?

Reply:
ups. A leftover from the quest for the lock-file problem

Reply:
it's corrected